be/src/exprs/vectorized/json_functions.cpp

```diff
@@ -224,6 +224,14 @@ ColumnPtr JsonFunctions::get_json_string(FunctionContext* context, const Columns
     return _json_string_unescaped(context, Columns{jsons});
 }
 
+ColumnPtr JsonFunctions::get_json_object(FunctionContext* context, const Columns& columns) {
```

Comment:
Is this jst an alias of `get_json_string`? How about calling `get_json_string` directly in the `get_json_object`?